### PR TITLE
fix: another patch for tb-profile-plugin

### PR DESCRIPTION
**Pull request by azhou-determined**
## Checklist

- [ ] Bump VERSION to make the pushed images are tagged with the right version.
- [ ] Licenses should be included for new code which was copied and/or modified from any external code.

### CODE REVIEW

File: dockerfile_scripts/torch_tb_profiler_patch.patch [hunk 1]
```diff
@@ -13,4 +13,9 @@ index fa15705..e8a2420 100644
 +
                  except Exception as ex:
                      logger.warning("Failed to scan runs. Exception=%s", ex, exc_info=True)
++                    if not self._is_active:
```
Comment (ioga) [inline dockerfile_scripts/torch_tb_profiler_patch.patch:16]: perhaps a `finally: ` clause will be more appropriate in this try, whatever it's doing?

**Review by ioga — APPROVED** [review]

File: dockerfile_scripts/torch_tb_profiler_patch.patch (hunk 1 above)
Comment (azhou-determined) [reply dockerfile_scripts/torch_tb_profiler_patch.patch:17]: done

Comment (ioga) [reply dockerfile_scripts/torch_tb_profiler_patch.patch:17]: I believe the upper if part can be removed then?

Comment (azhou-determined) [reply dockerfile_scripts/torch_tb_profiler_patch.patch:17]: not quite. there's no exception in that case. it just runs through the for loop, doesn't find anything, and will try again until the end of time.

**Review by ioga — APPROVED** [review]

**Pull request merged**